### PR TITLE
TESTING [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by neuromusic**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -31,6 +31,8 @@ starfish: scalable pipelines for image-based transcriptomics
 
 -------------------------------
 
+SILLY CHANGE
+
```
Comment (ttung) [inline README.rst:35]: omg

Comment (neuromusic) [reply README.rst:35]: lol I was demonstrating how CI works for Lucy :P

Comment (ttung) [reply README.rst:35]: Deep.

Comment (dganguli) [reply README.rst:35]: you rang?

Comment (shanaxel42) [reply README.rst:35]: We miss you.

Comment (ttung) [reply README.rst:35]: AWW DEEP

**Pull request closed without merging**